e-shop-website-with-ilb-ase/azuredeploy.json

```diff
@@ -0,0 +1,888 @@
+            },
+            {
+              "name": "RedisConnection",
+              "type": "Custom",
+              "connectionString": "[concat(variables('redisCacheName'), '.redis.cache.windows.net:6380,password=', listKeys(resourceId('Microsoft.Cache/Redis/', variables('redisCacheName')), '2016-04-01').primaryKey, ',ssl=True,abortConnect=False')]"
+            }
+          ]
+        },
+        "hostNames": [
+          "[variables('webAppExternalHostName')]",
+          "[variables('webAppInternalHostName')]"
+        ]
+      },
+      "resources": [
+        {
+          "apiVersion": "2015-08-01",
+          "name": "web",
+          "type": "sourcecontrols",
+          "properties": {
+            "RepoUrl": "[variables('sourceCodeRepositoryURL')]",
+            "branch": "master",
+            "IsManualIntegration": true
+          },
+          "dependsOn": [
+            "[resourceId('Microsoft.Web/sites/', variables('webAppName'))]"
+          ]
+        }
+      ],
+      "dependsOn": [
+        "[resourceId('Microsoft.Web/serverfarms/', variables('aspName'))]",
+        "[resourceId('microsoft.insights/components/', variables('appInsightsName'))]",
+        "[resourceId('Microsoft.Storage/storageAccounts', variables('storageAccountName'))]",
+        "[resourceId('Microsoft.Cache/Redis/', variables('redisCacheName'))]",
+        "[resourceId('Microsoft.Web/sites/', variables('apiAppName'))]"
+      ]
+    },
+    {
+      "apiVersion": "2016-03-01",
+      "type": "Microsoft.Web/sites",
+      "name": "[variables('contentAppName')]",
+      "location": "[variables('location')]",
+      "properties": {
+        "name": "[variables('contentAppName')]",
+        "serverFarmId": "[resourceId('Microsoft.Web/serverfarms/', variables('aspName'))]",
+        "hostingEnvironment": "[parameters('existingAseName')]",
+        "siteConfig": {
+          "alwaysOn": true,
+          "appSettings": [
+            {
+              "name": "project",
+              "value": "src/Web/wwwroot"
+            }
+          ]
+        },
+        "hostNames": [
+          "[variables('contentAppExternalHostName')]",
+          "[variables('contentAppInternalHostName')]"
+        ]
+      },
+      "resources": [
+        {
+          "apiVersion": "2015-08-01",
+          "name": "web",
+          "type": "sourcecontrols",
+          "properties": {
+            "RepoUrl": "[variables('sourceCodeRepositoryURL')]",
+            "branch": "master",
+            "IsManualIntegration": true
+          },
+          "dependsOn": [
+            "[resourceId('Microsoft.Web/sites/', variables('contentAppName'))]"
+          ]
+        }
+      ],
+      "dependsOn": [
+        "[resourceId('Microsoft.Web/serverfarms/', variables('aspName'))]"
+      ]
+    },
+    {
+      "apiVersion": "2016-03-01",
```

Comment:
update apiVersion - do this throughout (won't comment on the rest)

Reply:
all updated